rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -126,7 +126,7 @@ def _insert_samples(self, participant, tests, sample_ids, identifier, confirmed_
       self.sample_dao.insert(BiobankStoredSample(
           biobankStoredSampleId=sample_id,
           biobankId=participant.biobankId,
-          biobankOrderIdentifier=identifier,
+          biobankOrderIdentifier=identifier + sample_id,
```

Comment:
This will make every sample have a different order identifier... more realistic will be having the values of identifier passed in vary, sometimes being the same for multiple samples... they should mirror the order IDs created for each participant.

(Again if you want, you can deal with it in the next PR.)